pyproject.toml

```diff
@@ -93,10 +93,14 @@ pyvrp = "pyvrp.cli:main"
 line-length = 79
 
 
-[tool.isort]
-case_sensitive = true
-line_length = 79
-profile = "black"
+[tool.ruff]
+ignore-init-module-imports = true
+line-length = 79
+select = ["E", "F", "I", "NPY", "RET", "RSE", "RUF", "SLF", "SIM", "TCH"]

```

Comment:
We might expand this list later. For now I've taken the defaults (E and F) and added a bunch of additional ones that seemed useful.